### DISCUSSION TITLE
Can Open-WebUI run hugging face models (non gguf)

### DISCUSSION THREAD

**Discussion by Immortality-IMT** [open-webui/open-webui#6280] [Q&A]
How to use open-webui to run a hugging face model, not via ollama or gguf format but this format... 
Perhaps some hack script. Many models do not convert to gguf.
I am thinking git clone the model from hugging face and then add it, hack it into to open-webui somewhere.

.gitattributes
README.md
config.json
generation_config.json
model-00001-of-00009.safetensors
model-00002-of-00009.safetensors
model-00003-of-00009.safetensors
model-00004-of-00009.safetensors
model-00005-of-00009.safetensors
model-00006-of-00009.safetensors
model-00007-of-00009.safetensors
model-00008-of-00009.safetensors
model-00009-of-00009.safetensors
model.safetensors.index.json
special_tokens_map.json
tokenizer.json
tokenizer.model
tokenizer_config.json

**Comment by illtellyoulater** [comment]
did you figure this out?

**Reply by illtellyoulater** [reply]
I did myself :)
Check  out https://github.com/ggerganov/llama.cpp/discussions/8532#discussioncomment-11899485

**Comment by zveroboy152** [comment]
I'm looking to be able to do this too.  Any ideas on how to get this working?

**Reply by illtellyoulater** [reply]
I managed to convert it to gguf, see https://github.com/ggerganov/llama.cpp/discussions/8532#discussioncomment-11899485

**Reply by Immortality-IMT** [reply]
The issue is many of these models of not convertible to ggnuf
